Microsoft.HealthVault/Connection/HealthVaultConnectionBase.cs

```diff
@@ -178,6 +178,11 @@ protected HealthVaultConnectionBase(IServiceLocator serviceLocator)
             this.SessionCredential = await sessionCredentialClient.GetSessionCredentialAsync(token).ConfigureAwait(false);
         }
 
+        Task IConnectionInternal.RefreshSessionCredentialAsync(CancellationToken token)
+        {
+            return this.RefreshSessionCredentialAsync(token);
```

Comment:
This just calls the method above

---
In reply to: [115799063](https://github.com/Microsoft/healthvault-dotnetstandard-sdk/pull/27#discussion_r115799063) [](ancestors = 115799063)